src/rust/madara/sources/mangaread/res/filters.json

```diff
@@ -0,0 +1,315 @@
+				"name": "Historical",
+				"id": "historical"
+			},
+			{
+				"type": "genre",
+				"name": "Horror",
+				"id": "horror"
+			},
+			{
+				"type": "genre",
+				"name": "Isekai",
+				"id": "isekai"
+			},
+			{
+				"type": "genre",
+				"name": "Josei",
+				"id": "josei"
+			},
+			{
+				"type": "genre",
+				"name": "Magic",
+				"id": "magic"
+			},
+			{
+				"type": "genre",
+				"name": "Live action",
+				"id": "live-action"
+			},
+			{
+				"type": "genre",
+				"name": "Manga",
+				"id": "manga"
+			},
+			{
+				"type": "genre",
+				"name": "Manhua",
+				"id": "manhua"
+			},
+			{
+				"type": "genre",
+				"name": "Manhwa",
+				"id": "manhwa"
+			},
+			{
+				"type": "genre",
+				"name": "Martial Arts",
+				"id": "martial-arts"
+			},
+			{
+				"type": "genre",
+				"name": "Mature",
+				"id": "mature"
+			},
+			{
+				"type": "genre",
+				"name": "Mecha",
+				"id": "mecha"
+			},
+			{
+				"type": "genre",
+				"name": "Military",
+				"id": "military"
+			},
+			{
+				"type": "genre",
+				"name": "Mystery",
+				"id": "mystery"
+			},
+			{
+				"type": "genre",
+				"name": "One shot",
+				"id": "one-shot"
+			},
+			{
+				"type": "genre",
+				"name": "Psychological",
+				"id": "psychological"
+			},
+			{
+				"type": "genre",
+				"name": "Reincarnation",
+				"id": "reincarnation"
+			},
+			{
+				"type": "genre",
+				"name": "Romance",
+				"id": "romance"
+			},
+			{
+				"type": "genre",
+				"name": "School Life",
+				"id": "school-life"
+			},
+			{
+				"type": "genre",
+				"name": "Sci-fi",
+				"id": "sci-fi"
+			},
+			{
+				"type": "genre",
+				"name": "Seinen",
+				"id": "seinen"
+			},
+			{
+				"type": "genre",
+				"name": "Shoujo",
+				"id": "shoujo"
+			},
+			{
+				"type": "genre",
+				"name": "Shoujo Ai",
+				"id": "shoujo-ai"
+			},
+			{
+				"type": "genre",
+				"name": "Shounen",
+				"id": "shounen"
+			},
+			{
+				"type": "genre",
+				"name": "Shounen Ai",
+				"id": "shounen-ai"
+			},
+			{
+				"type": "genre",
+				"name": "Slice of Life",
+				"id": "slice-of-life"
+			},
+			{
+				"type": "genre",
+				"name": "Smut",
+				"id": "smut"
+			},
+			{
+				"type": "genre",
+				"name": "Soft Yaoi",
+				"id": "soft-yaoi"
+			},
+			{
+				"type": "genre",
+				"name": "Soft Yuri",
+				"id": "soft-yuri"
+			},
+			{
+				"type": "genre",
+				"name": "Sports",
+				"id": "sports"
+			},
+			{
+				"type": "genre",
+				"name": "Super Power",
+				"id": "super-power"
+			},
+			{
+				"type": "genre",
+				"name": "Supernatural",
+				"id": "supernatural"
+			},
+			{
+				"type": "genre",
+				"name": "Thriller",
+				"id": "thriller"
+			},
+			{
+				"type": "genre",
+				"name": "Tragedy",
+				"id": "tragedy"
+			},
+			{
+				"type": "genre",
+				"name": "Webtoon",
+				"id": "webtoon"
+			},
+			{
+				"type": "genre",
+				"name": "Yaoi",
+				"id": "yaoi"
+			},
+			{
+				"type": "genre",
+				"name": "Yuri",
+				"id": "yuri"
+			}
+		]
+	}
+]
```

Comment:
```suggestion
]

```

Reply:
json needs a newline

Reply:
When I try to commit your suggestion it doesn't works![image](https://github.com/Skittyblock/aidoku-community-sources/assets/43102748/cab008bc-6c7f-4cb2-8fa2-85974fa50d2a)